app/Console/Commands/FetchCollectionActivity.php

```diff
@@ -45,7 +45,7 @@ public function handle(): int
         };
 
         $this->forEachCollection(function ($collection) {
-            Job::dispatch($collection);
+            Job::dispatch($collection, forced: true);
```

Comment:
what does this do?

Reply:
when stress-testing, i discovered that there is a bug with the job... which is if the job is rate limited on the very first attempt, it will be released back onto the queue, but since the `is_fetching_activity` flag has been set and `forced` is `false`, it will not run, being left in this weird state where job didn't run but `is_fetching_activity` is true.

so i set the forced flag to make sure it runs if rate limited on the very first attempt